design/simple-client-configuration.md

```diff
@@ -0,0 +1,2085 @@
+
+-   Kubernetes platform is available with \`kubectl\` and \`helm\` local
+    test client access.
+
+-   Conjur instance is available. Testing should support the following
+    three configurations:
+
+    -   Conjur OSS in the Kubernetes cluster
+
+    -   Conjur Enterprise master outside of Kubernetes cluster,
+        followers inside the Kubernetes cluster
+
+    -   Conjur Enterprise master and followers outside of Kubernetes
+        cluster
+
+## Test cases 
+
+### Helm lint tests for Kubernetes cluster prep Helm chart
+
+Running ‘helm lint’ on the Kubernetes cluster prep Helm chart will validate
+the structure of templates contained in the Helm chart. This can be tested
+either in a Jenkins pipeline or with the ‘[helm-check](https://github.com/marketplace/actions/github-action-for-helm-templates-validation)’
+GitHub action.
+
+### Helm lint tests for application Namespace prep Helm chart
+
+Running ‘helm lint’ on the application Namespace prep Helm chart will
+validate the structure of templates contained in the Helm chart. This can
+be tested either in a Jenkins pipeline or with the ‘[helm-check](https://github.com/marketplace/actions/github-action-for-helm-templates-validation)’
+GitHub action.
+
+### Unit tests for Kubernetes cluster prep Helm chart
+
+These unit tests make use of the Helm chart [unittest project](https://github.com/lrills/helm-unittest#helm-unittest).
+
+“Negative” test cases are included to verify that the JSON schema validation
+for this chart fails as expected when configured chart values (e.g., For
+Conjur URL, Conjur SSL certificate, ServiceAccount name, etc.) are invalid. 
+
+<table>
+<thead>
+<tr class="header">
+<th></th>
+<th><strong>Scenario</strong></th>
+<th><strong>Status</strong><br />
+<strong>(Done / Not Done)</strong></th>
+<th><strong>Notes</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>1</td>
+<td>Default value for Conjur account used when not explicitly configured.</td>
+<td></td>
+<td></td>
+</tr>
+<tr class="even">
+<td>2</td>
+<td><p>Happy path: With valid chart values, templates for the following objects are created with the expected names:</p>
+<ul>
+<li><p>Golden ConfigMap</p></li>
+<li><p>Authenticator ServiceAccount</p></li>
+<li><p>Authenticator ClusterRole</p></li>
+</ul></td>
+<td></td>
+<td>NOTE 1, NOTE 2, NOTE 3, NOTE 4, NOTE 5, NOTE 6</td>
+</tr>
+<tr class="odd">
+<td>3</td>
+<td>With invalid Conjur URL, template rendering fails</td>
+<td></td>
+<td>NOTE 7, NOTE 8</td>
+</tr>
+<tr class="even">
+<td>4</td>
+<td>With invalid Conjur SSL certificate, template rendering fails</td>
+<td></td>
+<td>NOTE 9</td>
+</tr>
+<tr class="odd">
+<td>5</td>
+<td>With invalid authenticator ServiceAccount name, template rendering fails</td>
+<td></td>
+<td>NOTE 10</td>
+</tr>
+<tr class="even">
+<td>6</td>
+<td>With invalid authenticator Namespace name, template rendering fails</td>
+<td></td>
+<td>NOTE 11</td>
+</tr>
+</tbody>
+</table>
+
+<table>
+<thead>
+<tr class="header">
```

Comment:
This table should skip the header row, so e.g. this should be `<tr class="odd">` or something like that?

Reply:
If it's easier to use a simple list for these Notes rather than a table, that wold be fine by me!

Reply:
it would be easier if it weren't already a table :) fixing in 1 min...